app/controllers/admin/configs_controller.rb

```diff
@@ -220,5 +223,11 @@ def job_params
         display_jobs_banner
       ]
     end
+
+    def allowed_empty_enumerables
```

Comment:
Seems like this could be a constant?

Reply:
Great suggestion, they've been constantized!